pydevices/HtsDevices/acq2106_435st.py

```diff
@@ -240,8 +240,8 @@ def lcma(arr):
                 if nacc_str == '0,0,0':
                     nacc_sample = 1
                 else:
-                    nacc_tuple = ast.literal_eval(nacc_str)
-                    nacc_sample = nacc_tuple[0]
+                    nacc_sample = int(nacc_str.split(",")[0])
```

Comment:
We might want better error checking around this, in case the value isn't what we expect

Reply:
The error checking is done in the INIT method: there are checks to be sure that the values are in a range.